ledger/api-server-damlonx/reference-v2/src/test/suite/scala/com/daml/ledger/api/server/damlonx/reference/v2/InMemoryKVParticipantStateIT.scala

```diff
@@ -1,10 +1,11 @@
 // Copyright (c) 2019 The DAML Authors. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.ledger.participant.state.kvutils
+package com.daml.ledger.api.server.damlonx.reference.v2
```

Comment:
We may want to simplify this package name by e.g. removing `damlonx` from its name.

Reply:
Agreed, but I didn't add this package so I'm inclined to make this separate (and leave it to you to do it 😉 ).